frontend/util/city-input.component.tsx

```diff
@@ -47,7 +47,7 @@ export default function CityInput(props) {
         onChange={evt => props.setCity(evt.target.value)}
         required
         ref={inputRef}
-        autoComplete="off"
+        autoComplete="new-password"
```

Comment:
should we do this on all the `<form>`s that we have? Demographics, Services, Client Source, etc?

Reply:
I think I've found all the spots it was needed in the add client section now